### PR TITLE
Exclude non front panel interfaces from nexthop

### PR DESCRIPTION
**Pull request by qiluo-msft**
Following https://github.com/Azure/sonic-snmpagent/pull/29, fix a syntax bug.

### CODE REVIEW

File: src/sonic_ax_impl/mibs/ietf/rfc4292.py [hunk 1]
```diff
@@ -35,7 +35,12 @@ def update_data(self):
                 ipn = ipaddress.ip_network(ipnstr)
                 ent = self.db_conn.get_all(mibs.APPL_DB, routestr, blocking=True)
                 nexthops = ent[b"nexthop"].decode()
-                for nh in nexthops.split(','):
+                ifnames = ent[b"ifname"].decode()
+                for nh, ifn in zip(nexthops.split(','), ifnames.split(',')):
```
Comment (stcheng) [inline src/sonic_ax_impl/mibs/ietf/rfc4292.py]: according to my observation, it is possible that the number of nexthops and the number of ifnames cannot match. it happens when the ifnames is `eth0` but no nexthops.
```
127.0.0.1:6379> hgetall ROUTE_TABLE:10.3.146.0/23
1) "nexthop"
2) ""
3) "ifname"
4) "eth0"
```

Comment (qiluo-msft) [inline src/sonic_ax_impl/mibs/ietf/rfc4292.py]: The code here tolerates this case. zip() just returns a list as long as the shorter one of nexthops and ifnames.

Comment (stcheng) [inline src/sonic_ax_impl/mibs/ietf/rfc4292.py]: if it shortens the input, then the above case will return empty list I think

Comment (stcheng) [inline src/sonic_ax_impl/mibs/ietf/rfc4292.py]: I see. no problem here.

**Pull request merged**